invenio_search_ui/templates/invenio_search_ui/search.html

```diff
@@ -122,13 +122,19 @@
       {%- endblock search_pagination_bottom %}
     </div>
     {%- endblock search_body %}
-    {%- block search_facets %}
     <div class="col-md-3 col-md-pull-9 col-xs-12">
+      {%- block search_range %}
+      <invenio-search-range
+       options='{{ config.SEARCH_UI_JSTEMPLATE_RANGE_OPTIONS | tojson }}'
+       template="{{ url_for('static', filename=config.SEARCH_UI_JSTEMPLATE_RANGE) }}">
+      </invenio-search-range>
+      {%- endblock search_range %}
+      {%- block search_facets %}
```

Comment:
You should not change the location of the block. Please move it back to line 125.